### DISCUSSION TITLE
Modify/personalize icon on <Rating>

### DISCUSSION THREAD

**Discussion by AECSTacconi** [BlazorFluentUI/BlazorFluentUI#418] [Q&A]
Hi,
there is a way to usa different icon on rating using the attribute "IconName" or at least modify the color of the star using "Style" attribute?

Thanks

**Comment by vnbaaij** [comment]
Hi,

I suppose it should be possible to change the color with some css styling. It is however a general accepted way to have ratings in yellow stars. Would think twice to change that.
This repository is not actively being worked on anymore. Consider changing to the Microsoft Fluent UI Blazor component library. Thanks.
